src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -444,6 +485,22 @@ private boolean isCreateAccountBasedOnEmail() {
         return descriptor.isCreateAccountBasedOnEmail();
     }
 
+    private boolean isUseExistingAccountWithSameEmail() {
+        DescriptorImpl descriptor = getGitSCMDescriptor();
+
+        if (descriptor == null) {
+            return false;
+        }
+
+        return descriptor.isUseExistingAccountWithSameEmail();
+    }
+
+    @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
+        justification="Jenkins.getInstance() is not null")
+    private DescriptorImpl getGitSCMDescriptor() {
+        return (DescriptorImpl) Jenkins.getInstance().getDescriptor(GitSCM.class);
```

Comment:
Note for @MarkEWaite: If the plugin would require a newer Jenkins LTS version (e.g. 2.121.3) there would be `Jenkins::get` which is annotated with `@Nonnull`. Therefore the `@SuppressFBWarnings` annonation would not be required anymore.

Reply:
Done